src/lib/utils.ts

```
@@ -161,6 +161,13 @@ export const getContentDataFromCreateTransaction = (
   return contentData;
 };
 
+export const getPaymentDataFromCreateTransaction = (
+  createParameters: RequestLogicTypes.ICreateParameters,
+) => {
+  const extensionData = createParameters?.extensionsData!;
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)